src/reader.ts

```diff
@@ -116,16 +93,13 @@ async function getDependencies(
  */
 async function getContent(
   location: string,
-  token?: string,
-  group?: string,
-  name?: string,
-  branch?: string
+  opts?: ReaderOpts
```

Comment:
I agree the `opts` can be optional, but... the ReaderOpts attributes should be mandatory, I suggest
```
export type ReaderOpts = {
  token: string,
  group: string,
  name: string,
  branch: string
}
```
[reader.test.ts](https://github.com/kiegroup/build-chain-configuration-reader/pull/55/files#diff-e33c6aa0c10ee713f2f749b91c9e4cd4c8ae3d520a35527fd9fb2adae0751d65) should be adapted

Reply:
Well you might not necessarily need all of them no? For example:
Example 1: Just need the token and nothing else
Example 2: Just need the name and branch and we want to use the default value of group